modules/nf-neuro/betcrop/antsbet/main.nf

```diff
@@ -19,22 +19,26 @@ process BETCROP_ANTSBET {
     task.ext.when == null || task.ext.when
 
     script:
-    def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def args = []
+    if (mask) args += ["-f $mask"]
+    if (initial_affine) args += ["-r $initial_affine"]
 
     """
-    export ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS=1
+    export ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS=$task.cpus
     export OMP_NUM_THREADS=1
     export OPENBLAS_NUM_THREADS=1
     export ANTS_RANDOM_SEED=1234
 
-    antsBrainExtraction.sh -d 3 -a $t1 -e $template -o bet/ -m $tissues_probabilities -u 0
-    scil_volume_math.py convert bet/BrainExtractionMask.nii.gz ${prefix}__t1_bet_mask.nii.gz --data_type uint8
-    mrcalc $t1 ${prefix}__t1_bet_mask.nii.gz -mult ${prefix}__t1_bet.nii.gz -nthreads 1
+    antsBrainExtraction.sh -d 3 -a $t1 -o bet/ -u 0 \
+        -e $template -m $tissues_probabilities ${args.join(' ')}
+    scil_volume_math.py convert bet/BrainExtractionMask.nii.gz \
+        ${prefix}__t1_bet_mask.nii.gz --data_type uint8
+    mrcalc $t1 ${prefix}__t1_bet_mask.nii.gz -mult ${prefix}__t1_bet.nii.gz -nthreads $task.cpus
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        scilpy: 2.0.0
+        scilpy: \$(pip list | grep scilpy | tr -s ' ' | cut -d' ' -f2)
         mrtrix: \$(mrcalc -version 2>&1 | sed -n 's/== mrcalc \\([0-9.]\\+\\).*/\\1/p')
         ants: 2.4.3
```

Comment:
Can you add a dynamic version for `ANTs` ? Something like that should do the trick: `antsRegistration --version | grep "Version" | sed -E 's/.*v([0-9]+\\.[0-9]+\\.[0-9]+).*/\\1/'`